ios/PurchasesHybridCommon/PurchasesHybridCommonTests/PurchasesHybridCommonTests.swift

```diff
@@ -37,45 +37,45 @@ class PurchasesHybridCommonTests: QuickSpec {
         }
 
         context("logIn") {
-            it("passes the call correctly to Purchases") {
-                let mockPurchases = MockPurchases()
-                let mockPurchaserInfo = PartialMockPurchaserInfo()
-                let mockCreated = Bool.random()
-                mockPurchases.stubbedLogInCompletionResult = (mockPurchaserInfo, mockCreated, nil)
-
-                Purchases.setDefaultInstance(mockPurchases)
-
-                let appUserID = "appUserID"
-                RCCommonFunctionality.logIn(withAppUserID: appUserID) { _, _ in }
-
-                expect { mockPurchases.invokedLogInCount } == 1
-                expect { mockPurchases.invokedLogInParameters?.appUserID } == appUserID
-            }
-
-            it("returns purchaserInfo and created if successful") {
-                let mockPurchases = MockPurchases()
-                let mockPurchaserInfo = PartialMockPurchaserInfo()
-                let mockCreated = Bool.random()
-                mockPurchases.stubbedLogInCompletionResult = (mockPurchaserInfo, mockCreated, nil)
-
-                Purchases.setDefaultInstance(mockPurchases)
-                var receivedResultDict: NSDictionary?
-                var receivedError: ErrorContainer?
-
-                let appUserID = "appUserID"
-                RCCommonFunctionality.logIn(withAppUserID: appUserID) { resultDict, error in
-                    receivedResultDict = resultDict! as NSDictionary
-                    receivedError = error
-                }
-
-                let expectedResult: NSDictionary = [
-                    "purchaserInfo": mockPurchaserInfo.dictionary() as NSDictionary,
-                    "created": mockCreated
-                ]
-
-                expect { receivedResultDict } == expectedResult
-                expect { receivedError }.to(beNil())
-            }
+//            it("passes the call correctly to Purchases") {
```

Comment:
these were calling the methods that get temporarily removed, but we'll re-enable the tests after migrating to v4